.golangci.yaml

```diff
@@ -1,62 +1,65 @@
 ---
+version: "2"
 run:
-  # Tell golangci-lint to not acquire a lock because
-  # arcanist will run multiple instances in parallel.
   allow-parallel-runners: true
-  # arcanist runs many of these in parallel causing
-  # CPU contention and longer runtimes.
-  timeout: 3m
-
-output:
-  sort-results: true
-
-issues:
-  max-issues-per-linter: 0
-  max-same-issues: 0
-  # TODO(ddelnano): Remove once typecheck is upgraded in next golangci-lint upgrade
-  # This error originates from the stdlib due to generics usage
-  exclude-rules:
-  - path: .*slices\/sort.go
-    linters:
-    - typecheck
-    text: "^(undefined: (min|max))"
-
 linters:
   enable:
   - asciicheck
-  - errcheck
-  # Although goimports includes gofmt, it doesn't support the simplify option.
-  # So we include gofmt here.
-  - gofmt
-  - gosimple
-  - govet
-  - ineffassign
   - makezero
   - misspell
   - nakedret
   - nolintlint
   - nonamedreturns
   - predeclared
   - revive
-  - staticcheck
-  # https://github.com/golangci/golangci-lint/issues/2649
-  # - structcheck
-  - typecheck
-  - unused
-  # https://github.com/golangci/golangci-lint/issues/2649
-  # - wastedassign
   - whitespace
```

Comment:
how did you decide which ones to remove?
if that's just because those linters are default enabled, I'd recommend explicitly keeping them in this list anyway.

Reply:
The majority of the changes to this file were done via `golangci-lint migrate`. The only portions I edited manually were the [linter settings](https://github.com/pixie-io/pixie/pull/2169/files#diff-9917ddc9f1c3304218f7269265b746d997c5c0615478177b5fceecd33ef47cb5R18-R38)

Reply:
I've added in the others that were removed in 0bb0021ce. The only exceptions are `typecheck` and `gosimple`.  The former can't be enabled or disabled and the latter was made a part of `staticcheck`.